rfc/composite_table_availability.pseudocode

```diff
@@ -0,0 +1,116 @@
+
+PhysicalTableSchema:
+	getFieldName(Column c)
```

Comment:
Per side-discussion, I believe that Fieldname per the existing class references a column in the fact store (in this case, in Druid)